strapi/strapi
# chore(content-releases): remove future flags

### What does it do?

- Removes the future flag from content-releases
- Optionally chains .options.draftAndPublish

### Why is it needed?

- The feature is stable

### How to test it?

- Remove the features config for future.contentReleases
- Use a license that has the feature 
- Releases should function as normal
